tests/testthat/testSmallCosmicNCV.tsv

```diff
@@ -0,0 +1,3 @@
+Sample name	ID_SAMPLE	ID_NCV	zygosity	GRCh	genome position	Mutation somatic status	WT_SEQ	MUT_SEQ	SNP	FATHMM_MKL_NON_CODING_SCORE	FATHMM_MKL_NON_CODING_GROUPS	FATHMM_MKL_CODING_SCORE	FATHMM_MKL_CODING_GROUPS	Whole_Genome_Reseq	Whole_Exome	ID_STUDY	PUBMED_PMID
```

Comment:
This is not necessary and can be removed. NCV data consists of non coding variants, which you mentioned cannot move into our workflow.